docs/js.md

```diff
@@ -413,3 +390,117 @@ mdocVariables := Map(
   "js-batch-mode" -> "true"
 )
 ```
+### Remapping imports at link time.
+
+It may be the case, that the library you wish to depend on, is a facade to some NPM dependancy. Mapping the import offers an alterative to bundling, by resolving the dependancy out of a CDN. We set mdoc to output ESModules (see `ModuleKind`), and load the dependancies directly in the browser.
+
+To mdocs CLI, you'll need to pass this flag;
+
+```sh
+--import-map-path full/path/to/import-map.json
+```
+
+`import-map.json` could look like this;
+
+```json
+{
+  "imports":
+  {
+    "@shoelace-style/shoelace/dist/": "https://cdn.jsdelivr.net/npm/@shoelace-style/shoelace@2.13.1/cdn/"
+  }
+}
+```
+You should then have this dependency available to you.
+
+
+# Legacy
+
+This documentation is preserved here, but it is no longer recommend to work with scalaJS bundler. If you require a bundler you should consider the excellent [vite](https://vitejs.dev) project. It is currently unsolved how to use mdoc with vite.
```

Comment:
recommendED.

also, by "unsolved", do you mean that you don't have a specific recommendation on how to do it? otherwise, it sounds like people have tried using vite but couldn't figure it out, so it might be very hard.

Reply:
I mean that I don't know how to use vite with mdocJS. I suspect, that it may not be easy. I have not invested a significant amount of time trying as I don't believe (personally) in that direction of travel. There may be others who do, I don't believe this PR would impair them... it would be a step on that journey. 

I'll change the wording to "potential improvement"?